portfolio/src/main/webapp/index.html

```diff
@@ -105,6 +106,30 @@ <h1 class="about0" id="contact"> Contact me </h1>
             <a href="https://www.linkedin.com/in/tammy-lim-lee-xin-57890a161/" class="link2">LinkedIn</a>
         </div>
     </div>
+
+    <!-- Comments -->
+    <div>
+        <h1 class="about0 id="commentid"> Leave a comment! </h1>
+        
+    </div>
+
+    <div class="comment0">
+        <form action="/data" method="POST">
+            <textarea class="comment1" name="comment-input">Type your comment here...</textarea>
```

Comment:
let's give a more definitive class name, we can also name it `comment-input` instead of `comment1`. In the CSS, we can access them using `.comment-input`. A class is defined using a . and an ID is defined using a #. So as far as the browser is concerned, they're two separate items, so it's ok to have the same name for class and ID.